package.json

```diff
@@ -43,7 +43,7 @@
     "marko-components"
   ],
   "devDependencies": {
-    "@ebay/skin": "^3.3.0-0",
+    "@ebay/skin": "^3.4.0-0",
```

Comment:
You can change this to 3.4.0 now. We have verified it.

Reply:
okay I will rebase from 0.1.0